<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSDesign</class>
 <widget class="QWidget" name="DSDesign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSDesign</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_input">
      <attribute name="title">
       <string>输入界面 </string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>300</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>请输入要输入课程总数：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_classnum">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>230</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>300</y>
         <width>500</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>请输入课程序号、课程名、课程前序、课时：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_over_1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>360</y>
         <width>35</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>500</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>请输入学期总学时（要大于单个课程学时）：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_CTimes">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_classnum">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>230</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>课程输入数结束</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_over">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>360</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>  课程信息输入结束</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_toltimes">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>100</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>学期学时输入结束</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_over_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>360</y>
         <width>180</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_over_3">
       <property name="geometry">
        <rect>
         <x>325</x>
         <y>360</y>
         <width>35</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_over_4">
       <property name="geometry">
        <rect>
         <x>365</x>
         <y>360</y>
         <width>35</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>430</y>
         <width>500</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>请输入需要删除的输入条目序号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_delete">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>490</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_delete">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>490</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string> 删除并展示</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_inT">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>80</y>
         <width>50</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>20</y>
         <width>701</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>输入序号                        课程名                         课程序号  课程前序  课时：</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_cN">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>80</y>
         <width>460</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_Cno">
       <property name="geometry">
        <rect>
         <x>1370</x>
         <y>80</y>
         <width>50</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_cP">
       <property name="geometry">
        <rect>
         <x>1430</x>
         <y>80</y>
         <width>50</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_cT">
       <property name="geometry">
        <rect>
         <x>1500</x>
         <y>80</y>
         <width>50</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_sort">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>580</y>
         <width>320</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>信息录入</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_present">
      <attribute name="title">
       <string>展示界面 </string>
      </attribute>
      <widget class="QPushButton" name="btn_output">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>780</y>
         <width>320</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>输出课表</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>16</y>
         <width>122</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string> 第二学年</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>70</y>
         <width>300</width>
         <height>700</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>16</y>
         <width>122</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string> 第三学年</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_3">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>70</y>
         <width>300</width>
         <height>700</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_4">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>70</y>
         <width>300</width>
         <height>700</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>1020</x>
         <y>16</y>
         <width>136</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>  第四学年</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>16</y>
         <width>122</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string> 第一学年</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>300</width>
         <height>700</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>维护界面</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>400</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>请输入您想查找的课程：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_search">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>140</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_search">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>140</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>230</y>
         <width>400</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>请输入您想修改的课程：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_change_1">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>290</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>原名称</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_change">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>290</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_change_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>330</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>修改名称</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>120</y>
         <width>871</width>
         <height>381</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/background1.png);</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
